README.md

```diff
@@ -40,4 +42,25 @@ To create a new Component within that Module:
 - `cd newModuleName` to change directory into the newModule folder
 - `ng g component newComponentName` to create a component as a child of the module
 
-From there, you can implement your new module/component. Don't forget to wire up your component to the module, and finally wire up your module to your app. 
\ No newline at end of file
+From there, you can implement your new module/component. Don't forget to wire up your component to the module, and finally wire up your module to your app. 
+
+## Environment Variables
```

Comment:
Great additions! And concise summary. Environment Variables would make for a great talking point, too.